tests/generate_code.sh

```diff
@@ -57,6 +57,9 @@ $TEST_NOINCL_FLAGS $TEST_CPP_FLAGS $TEST_CS_FLAGS -o namespace_test namespace_te
 # Generate the schema evolution tests
 ../flatc --cpp --scoped-enums $TEST_CPP_FLAGS -o evolution_test ./evolution_test/evolution_v*.fbs
 
+# Generate kotlin code using @JvmStatic annotation.
+../flatc --kotlin --gen-jvmstatic -o gen_jvmstatic -I include_test monster_test.fbs
```

Comment:
This generates a ton of new files for such a small feature, lets not add this.

Reply:
@aardappel  I removed two lines of generate_code.sh(bat) includes generated kotlin files.